data/sles4sap/b1_log.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Since it's something that runs in the SUT and not in the worker, I think it's fine to have it shell. What it would be good is to add some comments and a summary to the shell-script so we can gather at a quick glance what is doing.

Reply:
Shell script comments added.